etcd/check.py

```diff
@@ -92,6 +93,18 @@ def check(self, instance):
         timeout = float(instance.get('timeout', self.DEFAULT_TIMEOUT))
         is_leader = False
 
+        # Gather self health status
+        health_state = AgentCheck.UNKNOWN
+        self_response = self._get_self_health(url, ssl_params, timeout)
+        if self_response is not None:
+            if 'health' in self_response:
+                state = AgentCheck.OK if self_response[key] == 'true' else AgentCheck.CRITICAL
+            else:
+                self.log.debug("Missing 'health' key in stats, can't determine health status.")
+
+        self.service_check(self.HEALTH_SERVICE_CHECK_NAME, state, tags=instance_tags+["url:{0}".format(url)])
```

Comment:
isn't the `url:` tag already in the `instance_tags` list (see code 10 lines above)